frontend/app/framework/utils/date-time.ts

```diff
@@ -6,9 +6,11 @@
  */
 
 import { addDays, addHours, addMilliseconds, addMinutes, addMonths, addSeconds, addYears, format, formatDistanceToNow, parse, parseISO, startOfDay, startOfMonth, startOfTomorrow, startOfWeek, startOfYesterday } from 'date-fns';
+import { enUS, nl } from 'date-fns/locale';
 import { DateHelper } from './date-helper';
 
 const DATE_FORMAT = 'yyyy-MM-dd';
+const LOCALES = { enUS, nl };
```

Comment:
Shall we not move this part to the date-helper as well? Resolving the date-fns locale...

Reply:
I think i did what you suggested.